.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -1,52 +1,48 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows a user to place an order and an admin to list it', () => {
-  cy.visit('/desktops/mac');
+it('allows a user to add an item and register', () => {
+  cy.visit('/en-gb/product/macbook');
   cy.contains('Add to Cart').click();
   cy.contains('You have added');
   cy.visit('/index.php?route=checkout/checkout');
-  cy.contains('Guest Checkout').click();
-  cy.get('#button-account').click();
-  cy.fixture('guests').then((guest) => {
-    cy.get('#input-payment-firstname').type(
-      `${guest.newGuest.firstName}.${random}`
+  cy.fixture('customers').then((customer) => {
+    cy.get('#input-firstname').type(
+      `${customer.newCustomer.firstName}.${random}`
     );
-    cy.get('#input-payment-lastname').type(
-      `${guest.newGuest.lastName}.${random}`
+    cy.get('#input-lastname').type(
+      `${customer.newCustomer.lastName}.${random}`
     );
-    cy.get('#input-payment-email').type(`${guest.newGuest.email}`);
-    cy.get('#input-payment-telephone').type(guest.newGuest.phone);
-    cy.get('#input-payment-address-1').type(`${guest.newGuest.address}`);
-    cy.get('#input-payment-city').type(guest.newGuest.city);
-    cy.get('#input-payment-postcode').type(guest.newGuest.postCode);
-    cy.get('#input-payment-zone').select(guest.newGuest.zone);
+    cy.get('#input-email').type(`${customer.newCustomer.email}`);
```

Comment:
We should randomize this email, it seems the app doesn't allow a second registration with the same email. We need to make sure our tests are auto-conclusive and repeatable.

Reply:
That's true, I will update it.